deps/build.jl

```diff
@@ -7,6 +7,13 @@ builddir = joinpath(BinDeps.depsdir(mpi), "build")
 prefix = joinpath(BinDeps.depsdir(mpi), "usr")
 src = joinpath(BinDeps.depsdir(mpi), "src")
 
+if isdir(builddir)
```

Comment:
I'm not worried about the particular code -- I was thinking about the `try ... catch nothing end` pattern only.